app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list):
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list):
+    return [elf.play_elf_song() for elf in elves]
```

Comment:
you should not return anything here. `play_elf_song()` returns `None`, so you just return a list of `None`s